IPython/html/static/widgets/js/widget.js

```diff
@@ -558,7 +611,7 @@ define(["widgets/js/manager",
             /**
              * Makes browser interpret a numerical string as a pixel value.
              */
-            if (/^\d+\.?(\d+)?$/.test(value.trim())) {
+            if (value && /^\d+\.?(\d+)?$/.test(value.trim())) {
```

Comment:
@jdfreder - can you look at this?  I was getting test failures on occasion because `value` was `undefined` or `null` (sorry, forget which).  I didn't trace down the real problem yet, though.